src/sass/skins/index.scss

```diff
@@ -5,32 +5,32 @@ Provides all MSL skin classes: partial styles for HTML elements
 .mw-brand-bg {
   background: $mw-bg-gradient-1;
   filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#0EBBC9', endColorstr='#5E46BD',GradientType=0 );
-  color: $mw-color-inverted;
+  color: $mw-neutral-text-inverted;
 }
 .mw-accent-bg {
   background: $mw-bg-gradient-2;
   filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#FF482A', endColorstr='#FFC255',GradientType=0 );
-  color: $mw-color-inverted;
+  color: $mw-neutral-text-inverted;
 }
 .mw-brand-underline {
   &::after {
     content: " ";
     display: block;
     height: 1px;
-    background: linear-gradient(90deg, $MW_COLOR_BRAND_1, $MW_COLOR_BRAND_2);
```

Comment:
Is mw-brand-underline used anywhere? In any case, gradients should be defined as variables also, just like solids.

Reply:
It all depends on if we still want to mantain `$MW_COLOR_BRAND_1` and `$MW_COLOR_BRAND_2`